corehq/apps/dashboard/templates/dashboard/base.html

```diff
@@ -206,5 +206,22 @@ <h1>
       </p>
     </div>
   {% endif %}
+  {% if request|ui_notify_enabled:"APP_USER_PROFILES_UPDATE" %}
```

Comment:
It'd be good to limit this to users who will find it most relevant. How about moving it to the mobile workers page?

Reply:
`ABILITY_TO_HIDE_TRANSLATIONS` is a notification that looks like it's just on a page, instead of pointing to the navigation.